combustion-android-ble/src/main/java/inc/combustion/framework/ble/device/NodeBleDevice.kt

```diff
@@ -77,31 +78,37 @@ internal class NodeBleDevice(
         }
     }
 
-    private fun observeUartResponses(callback: (suspend (responses: List<NodeUARTMessage>) -> Unit)? = null) {
+    private fun observeUartMessages(callback: (suspend (messages: List<NodeUARTMessage>) -> Unit)? = null) {
         uart.jobManager.addJob(
             key = uart.serialNumber,
             job = uart.owner.lifecycleScope.launch(
                 CoroutineName("UartResponseObserver"),
             ) {
                 uart.observeUartCharacteristic {data ->
-                    val responses = NodeUARTMessage.fromData(data.toUByteArray())
-                    callback?.invoke(responses)
+                    val messages = NodeUARTMessage.fromData(data.toUByteArray())
+                    callback?.invoke(messages)
```

Comment:
loos like most of the method can be skipped if `callback` is null

Reply:
Yeah, it can... I'm not sure why the callback is allowed to be null since it is only called from one place with a callback... I'll remove the `?`.